uber/models.py

```diff
@@ -927,7 +927,7 @@ def badge_cost(self):
         total = 0
         for attendee in self.attendees:
             if attendee.paid == c.PAID_BY_GROUP:
```

Comment:
Whoops, sorry about that; since I've opened like 10 pull requests tonight, I've been pretty careful about doing a ``git checkout master`` before making my changes so that every pull request doesn't have the contents of all of the other pull requests.  I guess I got sloppy and accidentally included the contents of #1312 here.  Alas.